classes/Tools.php

```diff
@@ -376,29 +376,19 @@ public static function getServerName()
      */
     public static function getRemoteAddr()
     {
-        if (function_exists('apache_request_headers')) {
-            $headers = apache_request_headers();
-        } else {
-            $headers = $_SERVER;
-        }
-
-        if (array_key_exists('X-Forwarded-For', $headers)) {
-            $_SERVER['HTTP_X_FORWARDED_FOR'] = $headers['X-Forwarded-For'];
+        if (!empty($_SERVER['HTTP_X_FORWARDED_FOR'])
```

Comment:
Why not using Symfony components? 
https://symfony.com/doc/current/create_framework/http_foundation.html